src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
@shyamnamboodiripad FYI, you may need to apply the same to the Eval projects.

Reply:
Ah thanks for the heads up! That probably explains why recent builds in #5873 have been failing...

Reply:
When I have cycles, I'll look into generalising MSBuild configs that are required for each AI project.